WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -173,12 +173,12 @@ public void onBindViewHolder(GridViewHolder holder, int position) {
             if (isLocalFile) {
                 loadLocalImage(media.getFilePath(), holder.imageView);
             } else {
-                WordPressMediaUtils.loadNetworkImage(getBestImageUrl(media), holder.imageView, mImageLoader);
```

Comment:
Now that the image loader isn't being passed, `mImageLoader` is unused and can be dropped.